packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -818,21 +891,22 @@ contract Deploy is Deployer {
     function transferProxyAdminOwnership() public broadcast {
         ProxyAdmin proxyAdmin = ProxyAdmin(mustGetAddress("ProxyAdmin"));
         address owner = proxyAdmin.owner();
-        address finalSystemOwner = cfg.finalSystemOwner();
-        if (owner != finalSystemOwner) {
-            proxyAdmin.transferOwnership(finalSystemOwner);
-            console.log("ProxyAdmin ownership transferred to: %s", finalSystemOwner);
+        address safe = mustGetAddress("Safe");
+        if (owner != safe) {
+            proxyAdmin.transferOwnership(safe);
+            console.log("ProxyAdmin ownership transferred to Safe at: %s", safe);
         }
     }
 
     /// @notice Transfer ownership of the DisputeGameFactory contract to the final system owner
     function transferDisputeGameFactoryOwnership() public onlyDevnet broadcast {
         DisputeGameFactory disputeGameFactory = DisputeGameFactory(mustGetAddress("DisputeGameFactoryProxy"));
         address owner = disputeGameFactory.owner();
-        address finalSystemOwner = cfg.finalSystemOwner();
-        if (owner != finalSystemOwner) {
-            disputeGameFactory.transferOwnership(finalSystemOwner);
-            console.log("DisputeGameFactory ownership transferred to: %s", finalSystemOwner);
+
+        address safe = mustGetAddress("Safe");
```

Comment:
This will not work in the case we are on a live network, ie we hit the branch where we did not deploy a safe

Reply:
Any suggestions for how that lack of this safe should be handled on a live network?

Reply:
I answered the above on how to populate the backend for `mustGetAddress`, it either reads from storage and then falls back to disk if its not found in storage

Reply:
I see. 
What I don't get though is how this script could work on a live deployment, unless we update the Safe helper functions to handle a threshold greater than 1.